interface/patient_file/summary/demographics_full.php

```diff
@@ -744,6 +737,12 @@ function end_group() {
 <!-- include support for the list-add selectbox feature -->
 <?php include $GLOBALS['fileroot']."/library/options_listadd.inc"; ?>
 
+<?php /*Include the validation script and rules for this form*/
+$form_id="DEM";
```

Comment:
This is used to bring the constraint for the fields in each form. 
We did some refactor of the submit me function and we took it out to another file that we are including. we are also using the same  function to do the query to bring the fields and the parameter is the form id so we are just defining it